sling/nlp/parser/ontonotes/annotations.py

```diff
@@ -201,6 +208,26 @@ def __init__(self, converter):
     # Current sentence index.
     self.sentence = None
 
+
+  # Generator that returns spans of specified type(s).
+  def spans(self, types_mask, label=False):
+    if types_mask & NER > 0:
+      for s in self.ner.spans:
+        yield (s, "NER") if label else s
```

Comment:
Now that you have defined constants for the span types (NER, PRED, ...), wouldn't it be easier to use these instead of the strings?